app/views/labelMap.scala.html

```diff
@@ -253,5 +252,17 @@
             $('.loader').fadeOut('slow');
         })
     });
+
+    // Fully destroy label modal when it's closed to prevent weird behavior when opening another
```

Comment:
Just add a period to the end of comments to fit the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide) :grin: